<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerwindow</class>
 <widget class="QMainWindow" name="customerwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>蛋壳行记</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabwidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>700</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>1000</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;我们&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 14pt &quot;等线&quot;;</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::East</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>关于我们</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>410</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/image/images/suite2.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>410</width>
        <height>300</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/image/images/single1.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>220</y>
        <width>540</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>440</y>
        <width>360</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>560</y>
        <width>220</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>180</y>
        <width>530</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;蛋壳行记致力于为所有旅行者提供最优质的居住服务&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>290</y>
        <width>610</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;无数的旅行者们通过蛋壳收获了最温暖的行程&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>400</y>
        <width>661</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;现在我们诚挚地欢迎你翻开这一本&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>60</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>多年来</string>
      </property>
     </widget>
     <widget class="Line" name="line_6">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>100</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>520</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>属于你自己的行记</string>
      </property>
     </widget>
     <widget class="Line" name="line_7">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>330</y>
        <width>460</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>560</y>
        <width>170</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Page 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
     <zorder>line_3</zorder>
     <zorder>label_6</zorder>
     <zorder>line</zorder>
     <zorder>line_2</zorder>
     <zorder>label_9</zorder>
     <zorder>label_10</zorder>
     <zorder>label_5</zorder>
     <zorder>line_6</zorder>
     <zorder>label_11</zorder>
     <zorder>line_7</zorder>
     <zorder>label_12</zorder>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>酒店列表</string>
     </attribute>
     <widget class="QPushButton" name="bshowH">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>20</y>
        <width>140</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>显示全部</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bIn">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>580</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>进入酒店</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lQueryH">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="hTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>940</width>
        <height>470</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="bQueryH1">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bQueryH2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>20</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;位置&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>70</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;名称&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>570</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Page 2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cQueryH">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;等线&quot;;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>选择地区</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>海淀区</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>朝阳区</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>丰台区</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>东城区</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>西城区</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>客房列表</string>
     </attribute>
     <widget class="QLabel" name="lFull">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>240</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;抱歉，您选择的客房余量为零&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bQueryR1">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>20</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bBook">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>580</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>预订客房</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bshowR">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>20</y>
        <width>140</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>显示全部</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="rTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>940</width>
        <height>470</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lQueryR">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bQueryR2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>20</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;类型&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>70</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;名称&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>570</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Page 3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cQueryR">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>170</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;等线&quot;;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>选择房间类型</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>单人间</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>大床房</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>标间</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>家庭房</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>套房</string>
       </property>
      </item>
     </widget>
     <zorder>bQueryR1</zorder>
     <zorder>bBook</zorder>
     <zorder>bshowR</zorder>
     <zorder>rTable</zorder>
     <zorder>lQueryR</zorder>
     <zorder>bQueryR2</zorder>
     <zorder>lFull</zorder>
     <zorder>label_20</zorder>
     <zorder>label_21</zorder>
     <zorder>label_17</zorder>
     <zorder>cQueryR</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>最近订单</string>
     </attribute>
     <widget class="QPushButton" name="bmyRefresh">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>400</y>
        <width>90</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bEnded">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>490</y>
        <width>90</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>退房</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bComment">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>540</y>
        <width>90</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>点评</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bOut">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>590</y>
        <width>90</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>离店</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bPayback">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>440</y>
        <width>90</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>退款</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lHName">
      <property name="geometry">
       <rect>
        <x>139</x>
        <y>400</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lRName">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>460</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lPrice">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>520</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lState">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>580</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>121</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;酒店名称&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>460</y>
        <width>121</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;客房名称&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>101</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;客房价格&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>580</y>
        <width>121</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;客房状态&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>930</width>
        <height>360</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/image/images/couple3.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>570</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Page 4&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>我的记录</string>
     </attribute>
     <widget class="QPushButton" name="bLogout">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>580</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>退出登录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bExit">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>580</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>退出系统</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="recordTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>940</width>
        <height>440</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="brr">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>580</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>540</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>540</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>9</y>
        <width>540</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;十分感谢你选择我们作为你的出行伙伴&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>70</y>
        <width>540</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;愿你接下来的每一次旅程都轻松愉快&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>560</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Page 5&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <zorder>bLogout</zorder>
     <zorder>bExit</zorder>
     <zorder>recordTable</zorder>
     <zorder>brr</zorder>
     <zorder>label_15</zorder>
     <zorder>label_16</zorder>
     <zorder>line_4</zorder>
     <zorder>line_5</zorder>
     <zorder>label_23</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
